gateway/enforcer/org.wso2.apk.enforcer/src/main/java/org/wso2/apk/enforcer/api/APIFactory.java

```diff
@@ -87,6 +89,7 @@ public void addApis(List<Api> apis) {
             logger.debug("Total APIs in new cache: {}", newApis.size());
         }
         this.apis = newApis;
+        CacheProviderUtil.cleanupCacheHolder(newApis);
```

Comment:
can't we do both cleanupCacheHolder and initializeOrgLevelCache same time as both involves getting the org list form apis and create/delete cache. I think we could do both around line 68 in this level. It would avoid per api org initialization check delay.